common/app/conf/switches/MonitoringSwitches.scala

```diff
@@ -42,6 +42,15 @@ trait MonitoringSwitches {
     exposeClientSide = true
   )
 
+  val CspReporting = Switch(
+    "Monitoring",
+    "csp-reporting",
+    "Enables logging of CSP violations",
+    safeState = Off,
+    never,
```

Comment:
Should we set a expiration date? so we are forced to reevaluate it in a set period of time in case we dropped the ball for whatever reason